documentation/api/command/v1/commands.markdown

```diff
@@ -153,6 +160,25 @@ The payload is expected to be a Puppet catalog, as a JSON object,
 conforming exactly to the [catalog wire format v6][catalogv6]. Extra
 or missing fields are an error.
 
+### "replace facts", version 4
+
+* Similar to version 6 of replace catalog, previously dashed fields are now
+  underscore-separated.
+
+* The `name` field has been renamed to `certname`, for consistency.
+
+See [fact wire format v4][factsv4] for more information on the
+payload of this command.
+
+### "store report", version 7
+
+* The nullable `catalog\_uuid`, `code\_id`, and `cached\_catalog\_status`
```

Comment:
I don't think these backslashed underscores render right on the site. Anywhere we have them in docs is an error.